<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>QinXiaofeng</author>
 <class>ArchInfoEditDialog</class>
 <widget class="QDialog" name="ArchInfoEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>183</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>文件目录修改</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>材料名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtClmc"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>形成时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtYear"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>年</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtMonth">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>月</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtDay">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>日</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>页数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtYs">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>备注：    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtremarks"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    border-image:url(:/image/blue_button_down.png);
    background-repeat: no-repeat;
}
QPushButton:hover
{
    border-image:url(:/image/grey_button_down.png);
    background-repeat: no-repeat;
}</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    border-image:url(:/image/blue_button_down.png);
    background-repeat: no-repeat;
}
QPushButton:hover
{
    border-image:url(:/image/grey_button_down.png);
    background-repeat: no-repeat;
}</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
